webapp/src/main/java/edu/si/trellis/CassandraApplication.java

```diff
@@ -87,6 +104,8 @@ public void importAndArrangeAdditionalConfig() {
         current().getContext().getPropertySources().stream().map(PropertySource::getName).forEach(log::debug);
         log.debug("Using Tamaya configuration:");
         log(current().getProperties());
+        acService = new WebACService(this.resourceService);
```

Comment:
I'd like to find some more flexible means of adding WebAC than hard-coding it, but even if we do, we should let CDI inject these guys into the fields. No need for `new`.

Reply:
Yes, I agree. I think I hesitated to push more CDI annotations upstream into trellis, but I needn't have.

Reply:
Firehose it with CDI.

Reply:
I don't think there is currently an abstraction with a noop implementation for either WebACFilter WebACService or BasicAuthFilter, which is why none of them are injected at this point. I could inject them as concrete class, of course, which might be better than calling "new". However, having modules register with the Application seems like a missing step in here. They need to become part of the singletons set and each of them may have some initialization to do..

Reply:
Well, no and yes. They may all have initialization to do, but we needn't register them via `getSingletons`. Inf act, ideally we would not register them actively at all-- they would be autodiscovered (and initialized) by CDI/JAX-RS, and I'm wondering if that's the missing ingredient here.